app/app/urls.py

```diff
@@ -0,0 +1,22 @@
+"""app URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.1/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import include, path
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('feedpeople/', include('feedpeople.urls')),
```

Comment:
Does Django require `feedpeople.urls` to be included here, even though we're not defining any urls for it?

Reply:
I agree with @jonseitz, I think we can exclude the URLs for all of these models unless we are creating a view.

Reply:
Good point, this was removed in ab4c24b.